app/models/register/error/ApiError.scala

```diff
@@ -16,72 +16,82 @@
 
 package models.register.error
 
-sealed trait RegisterError {
+sealed trait ApiError {
   val errorCode: String
   val errorMessage: String
 }
 
-object RegisterError {
+object ApiError {
```

Comment:
@iservport still thinking about do we need this ApiError model or not, can we discuss Monday morning